app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt

```diff
@@ -0,0 +1,99 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils
+
+import android.annotation.TargetApi
+import android.app.Activity
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.ContextWrapper
+import android.content.Intent
+import android.content.pm.PackageManager
+
+import android.os.Build
+import android.provider.Settings
+import android.support.annotation.VisibleForTesting
+
+import org.mozilla.focus.locale.Locales
+import org.mozilla.focus.session.SessionManager
+import org.mozilla.focus.session.Source
+
+import java.io.UnsupportedEncodingException
+import java.net.URLEncoder
+import java.util.Locale
+
+object SupportUtils {
+    val HELP_URL = "https://support.mozilla.org/kb/what-firefox-focus-android"
```

Comment:
These can be "const val". https://kotlinlang.org/docs/reference/properties.html#compile-time-constants